cerebrium/environments/custom-images.mdx

```diff
@@ -3,16 +3,23 @@ title: Custom Images
 description: Specify your versions, dependencies and packages to use
 ---
 
-By default, Cerebrium models are executed in Python 3.9 unless the Python version specified by you in your **config.yaml** is different. However, Cerebrium only supports version 3.9 and above.
+By default, Cerebrium models are executed in Python 3.9 unless the Python version specified by you in your **cerebrium.toml** is different. However, Cerebrium only supports version 3.9 and above.
 
-Traditionally, when working with Python, you will need access to Apt packages, Pip packages and Conda packages, and so we replicate this functionality as if you were developing locally.
-When creating your Cortex project, you can contain the following files
+Traditionally, when working with Python, you will need access to Apt packages, Pip packages and Conda packages. For a deployment to cerebrium, you can specify all of these in your **cerebrium.toml** file in the following locations:
 
-- **requirements.txt** - This is where you define your Pip packages.
-- **pkglist.txt** - This is where you can define Linux packages you would like to install. We run the apt-install command for items here.
-- **conda_pkglist.txt** - This is where you can define Conda packages you would like to install if you prefer using it for some libraries over pip. You can use both conda and pip in conjunction.
+- **[cerebrium.dependencies.pip]** - This is where you define your Pip packages in the form `module = version_constraints`. For example, `transformers = ">=4.35.0"` or `torch = "2.1.0"`
```

Comment:
Rather at this to the code snippet below so it reads nicely. Just replace this with a small paragraph saying "You define your pip and conda packages in the form. ..." Then code snippet is below for pip, conda and apt.

Also I see you have latest in some places, use that in examples since I didn't know that. Would have to keep specifying versions otherwise

Reply:
Done.
it can take either "" or "latest"